get-iap-token/src/main.ts

```diff
@@ -0,0 +1,24 @@
+import * as core from '@actions/core';
+import { GoogleToken } from 'gtoken';
+import { getCredentials } from './credentials';
+
+// Retrieve input.
+const IAPOAuthClientID = core.getInput('iap_oauth_client_id');
+const credentials = getCredentials();
```

Comment:
super nit: we seem to be following a convention where we grab all the inputs first. This way its easier to test helpers.